community/php7/APKBUILD

```diff
@@ -137,231 +143,244 @@ _build() {
 		--with-tidy=shared \
 		--enable-xml=shared \
 		--enable-xmlreader=shared \
+		--enable-xmlwriter=shared \
 		--with-xmlrpc=shared \
 		--with-xsl=shared \
 		--enable-wddx=shared \
 		--enable-zip=shared --with-libzip=shared \
 		$@ || return 1
-	sed -ri "s/^(EXTRA_LDFLAGS[ ]*\=.*)/\1 -lpthread/" Makefile  # see #183 (from main/php)
+	sed -ri "s/^(EXTRA_LDFLAGS[ ]*\=.*)/\1 -lpthread/" Makefile
 	make || return 1
 }
 
 build() {
 	cd "$builddir"
-
-	# phpdbg
+	# build phpdbg
 	_build --enable-phpdbg \
-		--enable-phpdbg \
 		--enable-phpdbg-webhelper \
 		--disable-cgi \
 		--disable-cli \
 		|| return 1
-	# apache2 module
+	# build apache2 module + recode extension
 	_build --disable-phpdbg \
 		--disable-cgi \
 		--disable-cli \
 		--with-apxs2 \
+		--with-recode=shared \
 		|| return 1
-	mv libs/libphp7.so sapi/apache2handler/mod_php7.so
-	# cgi,cli,fpm,embed,litespeed
+	mv libs/lib$pkgname.so sapi/apache2handler/
+	# build all other + imap extension
 	_build --disable-phpdbg \
 		--enable-fpm \
 		--enable-embed \
 		--with-litespeed \
+		--with-imap=shared --with-imap-ssl \
 		|| return 1
 }
 
 package() {
 	cd "$builddir"
-
 	make -j1 INSTALL_ROOT="$pkgdir" install || return 1
 
-	install -Dm644 php.ini-production "$pkgdir"/etc/$pkgname/php.ini || return 1
+	rm -fr "$pkgdir"/.[[:alpha:]]* || return 1
 
-	local file; for file in pear peardev pecl; do
-		sed -i -e "s~/usr/bin/php~/usr/bin/php7~g" \
-			-e "s~PHP=php~PHP=php7~" \
-			"$pkgdir"/usr/bin/$file || return 1
-	done
-	paxmark -m "$pkgdir"/usr/bin/php7
-
-	find "$pkgdir" -name '.*' | xargs rm -rf || return 1
-	rmdir "$pkgdir"/var/run
+	ln -sf php$_suffix "$pkgdir"/usr/bin/php
 }
 
 dev() {
 	default_dev
-	depends="$depends $pkgname $pkgname-pear"
 
 	mkdir -p "$subpkgdir"/usr/bin \
 		"$subpkgdir"/usr/lib/$pkgname
 
-	mv "$pkgdir"/usr/bin/php-config7 "$pkgdir"/usr/bin/peardev \
-		"$subpkgdir"/usr/bin/ || return 1
+	local file
+	for file in php-config phpize; do
+		mv "$pkgdir"/usr/bin/$file$_suffix \
+			"$subpkgdir"/usr/bin/ || return 1
+		ln -sf $file$_suffix "$subpkgdir"/usr/bin/$file
+	done
+
 	mv "$pkgdir"/usr/lib/$pkgname/build \
-		"$subpkgdir"/usr/lib/$pkgname/
+		"$subpkgdir"/usr/lib/$pkgname/ || return 1
```

Comment:
`|| return 1` is actually not needed here (it’s has zero effect, like writing `true || true`) since it’s the last statement of the function (implicit return).